pages/[slug].js

```diff
@@ -0,0 +1,54 @@
+import Head from "next/head";
+import MiscLayout from "../components/miscLayout";
+import { getAllPosts, getPostBySlug } from "../lib/pages";
+import { ReactMarkdown } from "react-markdown/lib/react-markdown";
+export default function Misc({ post }) {
+  return (
+    <MiscLayout>
+      <Head>
+        <title>PyCon India 2023, Hyderabad | {post.title}</title>
+      </Head>
+      <h1
+        className="com-head text-center"
+        data-aos="fade-down"
+        data-aos-duration="800"
+        id={post.slug}
```

Comment:
Please update the header file to use "code-of-conduct" as id for COC instead of "coc" since the slug is "code-of-conduct."

Reply:
It will not affect the navigation.

Reply:
It’s not affecting the navigation, it’s affecting the activation of nav item.(COC nav item)
You can test it.

Reply:
Updated as part of header update PR.